packages/lisk-core/.eslintrc.json

```diff
@@ -1,4 +1,7 @@
 {
-	"extends": ["lisk-base"],
-	"plugins": ["mocha"]
+	"extends": ["airbnb-base"],
```

Comment:
The reason is in lisk-base mocha rules were hardcoded. Since we switched to jest now the rules were conflicting.

We should decouple mocha rules from lisk-base configuration and have something like lisk/mocha, lisk/base.